internal/proxy/resource_rds_database.go

```diff
@@ -308,6 +334,7 @@ func (r *RDSDatabaseResource) Update(ctx context.Context, req resource.UpdateReq
 			Name:                      user.Name.ValueString(),
 			Username:                  user.UserName.ValueString(),
 			PasswordSecretsManagerArn: user.PasswordSecretsManagerARN.ValueString(),
+			DefaultLocalPort:          uint32(user.DefaultLocalPort.ValueInt64()),
```

Comment:
Because this field is optional we need to check if a value was provided and only set it if it was